server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy

```diff
@@ -209,11 +209,16 @@ class CreateController {
                 }
               }
 
-             /* if ( flash.error.size() == 0 ) {
-                flash.error.add("There has been an error creating the component. Please try again.")
-              }*/
+              if(result.errors.size() > 0){
+                flash.error = result.errors
+                println("Moe")
```

Comment:
Yay, the general developer eternalises himself!! :D

Reply:
Done